src/index.html

```diff
@@ -16,7 +37,496 @@
       defer
     ></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <!-- #endregion -->
+  <!-- #region header -->
+  <body class="page__body">
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-bar">
+            <a
+              class="top-bar__logo-link"
+              href="#"
+            >
+              <img
+                src="./images/header/themet.png"
+                alt="The MET logo"
+                class="top-bar__logo"
+              />
+            </a>
+            <div class="top-bar__icons">
+              <div
+                href="#"
+                class="icon icon--phone"
+              >
+                <div class="icon__contact-info">
+                  <div>
+                    The Met Fift Avenue
+                    <br />
+                    <a
+                      class="icon__contact-link"
+                      href="tel:+12125357710"
+                    >
+                      +1 212-535-7710
+                    </a>
+                  </div>
+                  <div>
+                    The Met Cloisters
+                    <br />
+                    <a
+                      class="icon__contact-link"
+                      href="tel:+12129233700"
+                    >
+                      +1 212-923-3700
+                    </a>
+                  </div>
+                </div>
+              </div>
+              <a
+                href="#menu"
+                class="icon icon--menu"
+              ></a>
+            </div>
+          </div>
+          <div class="header__bottom">
+            <h1 class="header__title">
+              Welcome
+              <br />
+              to The MET
```

Comment:
This header could be placed a little higher and font-size should be bigger on larger screens

Reply:
Done.